requirements/base.txt

```diff
@@ -105,6 +111,7 @@ django==4.2.15
     #   notifications-api-common
     #   open-api-framework
     #   zgw-consumers
+    #   zgw-consumers-oas
```

Comment:
wait why do we need this dep? I don't remember this being part of commonground-api-common